.gitignore

```diff
@@ -86,6 +86,10 @@ ENV/
 # Spyder project settings
 .spyderproject
 
+# IDE project files
+.idea/
+.vscode/
+
```

Comment:
Yes, that fine, but allows from other dev to make the same settings on his machine. Local `.gitignore` is more flexible  here as you can be sure that there will be no extra files in your repo.

Reply:
Expecting every project in the world to add ignore patterns for every IDE and tool in the world is not more flexible (but, seems to be what most people expect with Git). Global ignore all files for _your_ editors and you'll never commit them into another repo again.

Reply:
What I mean is that gitignore in a repo should be related to that repo. `.vscode` has nothing to do with the project, it's related to someone's computer, and therefore should be ignored on their computer (if VS code were smarter, it might add this to a global ignore when it was installing itself).

The gitignore files we have at work are typically about 2 lines long. They just contain something like `/dist` which is created by the build tool in our repo. More importantly, these changes have nothing to do with 'Feature/value checker', it can just be a little frustrating when people try to sneak in stylistic changes etc. to a project in a random PR.

Reply:
Removed from `.gitignore`